GitUI/CommandsDialogs/SettingsDialog/Pages/ColorsSettingsPage.cs

```diff
@@ -70,6 +75,33 @@ public bool UseSystemVisualStyle
             set => chkUseSystemVisualStyle.Checked = value;
         }
 
+        private bool SettingsAreModified
```

Comment:
Can this be tested?

Reply:
will make a separate PR to master branch